lib/exec.ts

```diff
@@ -91,6 +91,17 @@ export function executeDirect(
         if (command.startsWith('./')) command = path.join(process.cwd(), command);
     }
 
+    const unbufferPrefix = execProps<string>('unbufferStdout', undefined);
+    if (unbufferPrefix) {
+        const splitPrefix = unbufferPrefix.split(' '); // by default ['stdbuf', '-o0']
```

Comment:
Let's not split on spaces here: we have common-utils' `splitArguments` that handles quoting etc

```suggestion
        const splitPrefix = splitArguments(unbufferPrefix);
```